### ISSUE TITLE
Invalid Jinja2 syntax not detected (missing leading {)

### ISSUE THREAD

**Issue by c-po** [aristanetworks/j2lint#29]
The following is not detected:

```python3
{% if foo is defined %}
    bar-option
% else %}
    baz-option
{% endif %}
```

The missing leading `{` is not found.

**Comment by ClausHolbechArista** [comment]
The j2lint cannot detect this case, since it is simply not jinja. It will just be interpreted as a regular string, so ignored by the linter. We _could_ create a warning for closing tag (`%}`) with missing opening tag (`{%`) but the jinja2 engine accepts this and renders `% else %}` like any other string, so I am not sure if we want to enforce this.

**Comment by c-po** [comment]
A warning would probably be a good idea - and could be switched off like other linting rules.
